# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/cosmic-idle/hotfix-buildrs.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- /dev/null  2025-01-24 16:02:05.463333370 +0100
+++ cosmic-idle-epoch-1.0.0-alpha.5.1/build.rs  2025-01-24 16:02:42.680000040 +0100
@@ -0,0 +1,14 @@
+use std::process::Command;
+
+fn main() {
+    if let Some(output) = Command::new("git")
+        .args(&["rev-parse", "HEAD"])
+        .output()
+        .ok()
+    {
+        let git_hash = String::from_utf8(output.stdout).unwrap();
+        println!("cargo:rustc-env=GIT_HASH={}", git_hash);
+        println!("cargo:rustc-link-lib=xkbcommon");
+        println!("cargo:rustc-link-search=native=/usr/X11R7/lib64");
+    }
+}

